src/Microsoft.Tye.Core/Serialization/YamlSerializer.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using k8s;

```

Comment:
Does this class actually use the k8s OM for anything?

Reply:
No, removed.